pages/dropdown/focus-trap.page.tsx

```diff
@@ -39,44 +27,44 @@ export default function DropdownScenario() {
           <label>
```

Comment:
The in-page controls can be removed provided the context settings are open during the screenshot tests for better dev experience.